pkgs/development/compilers/gcc/4.8/default.nix

```diff
@@ -86,7 +86,9 @@ let majorVersion = "4";
         ../struct-ucontext-4.8.patch
         ../sigsegv-not-declared.patch
         ../res_state-not-declared.patch
-      ];
+      ]
+      # gcc-11 compatibility
+      ++ [ ../char-reload-4.patch ];
```

Comment:
```suggestion
        # gcc-11 compatibility
        (fetchpatch {
          name = "gcc4-char-reload.patch";
          url = "https://gcc.gnu.org/git/?p=gcc.git;a=commitdiff_plain;h=d57c99458933a21fdf94f508191f145ad8d5ec58";
          includes = [ "gcc/reload.h" ];
          sha256 = "sha256-66AMP7/ajunGKAN5WJz/yPn42URZ2KN51yPrFdsxEuM=";
        })
      ];
```

Reply:
Ah, so to clarify, we _can_ use fetchpatch here, as long as it isn't part of any platform's bootstrap chain, right?